src/SFA.DAS.Roatp.CourseManagement.Web/Models/Standards/StandardViewModel.cs

```diff
@@ -11,6 +11,12 @@ public class StandardViewModel
         public string CourseDisplayName { get; set; }
         public int LarsCode { get; set; }
         public string StandardUrl { get; set; }
+        public string Version { get; set; }
+        public string ApprovalBody { get; set; }
+
+        public bool ApprovalRequired => !string.IsNullOrEmpty(ApprovalBody) && !IsApprovedByRegulator.HasValue;
```

Comment:
This should be renamed to `ApprovalPending` which then reads well on the View where we show the link only if !Approval is pending" where required. 

Also `!string.IsNullOrEmpty(ApprovalBody)` should be a readonly property `IsRegulatedStandard` so it reads will here.